src/components/LeftSidebar/LeftSidebar.vue

```diff
@@ -20,14 +20,16 @@
 				<TransitionWrapper name="radial-reveal">
 					<!-- Filters -->
 					<NcActions v-show="searchText === ''"
-						:primary="isFiltered !== null"
+						:type="filters.length !== 0 ? 'secondary' : 'tertiary'"
```

Comment:
Maybe we keep `filters.length !== 0` as computed? It's repeated in 5 places
```suggestion
						:type="isFiltered ? 'secondary' : 'tertiary'"
```